cpg-core/src/main/kotlin/de/fraunhofer/aisec/cpg/helpers/functional/BasicLatticesRedesign.kt

```diff
@@ -27,6 +27,7 @@ package de.fraunhofer.aisec.cpg.helpers.functional
 
 import de.fraunhofer.aisec.cpg.graph.edges.flows.EvaluationOrder
```

Comment:
At some point we should rename this class, it sounds like it was a redesign some time ago but is now our established way of handling lattices?

Reply:
There's still the competing old implementation, so I'd keep it as-is until we finally get rid of that one